roles/redis_replication/tasks/main.yml

```diff
@@ -59,20 +59,21 @@
 - name: Open Port on FirewallD Public Zone
   ansible.posix.firewalld:
     port: "{{ redis_sentinel_port }}/tcp"
-    permanent: yes
+    permanent: true
     state: enabled
     zone: public
-    immediate: yes
+    immediate: true
   when:
     - ansible_facts.services["firewalld.service"] is defined
     - (ansible_facts.services["firewalld.service"].state == "running")
     - (ansible_facts.services["firewalld.service"].status == "enabled")
-  ignore_errors: true
 
 - name: Restart Redis
-  command: /bin/true
+  ansible.builtin.command: /bin/true
   notify: Restart Redis
+  changed_when: true # The task is always reported as changed
 
 - name: Restart Sentinel
-  command: /bin/true
-  notify: Restart Sentinel
\ No newline at end of file
+  ansible.builtin.command: /bin/true
+  notify: Restart Sentinel
+  changed_when: true # The task is always reported as changed
```

Comment:
Instead of using using changed_when: true, move to a notify/block section.  Something like:

```
- name: Configure Redis clustering
   notify:
     - Restart Redis
     - Restart Sentinel
   block:
     <rest of code>
```